server/routes/v1/tokenDiscovery.js

```diff
@@ -3,12 +3,23 @@ const rippled = require('../../lib/rippled');
 
 const log = require('../../lib/logger')({ name: 'token discovery' });
 
-const TIME_INTERVAL = 1000 * 60 * 5; // 5 minutes
+// whether this is running in the prod environment (or in dev/staging)
+// for the purpose of running locally, this equals false if the env var doesn't exist
+// DO NOT SET TO TRUE UNLESS YOU'RE SURE ABOUT BIGQUERY USAGE
+// aka don't let the site run after you're done using it, because it'll cost $$
+const IS_PROD_ENV = process.env.REACT_APP_MAINNET_LINK
+  ? process.env.REACT_APP_MAINNET_LINK.includes('xrpl.org')
+  : false;
+// how long the auto-caching should run in dev and staging environments
+// We want to turn it off after some time so it doesn't run when we don't need it, which costs us
+// money per BigQuery query
```

Comment:
Comment style nit: I think it's nice when comments follow normal sentence capitalization (So I think 'How' should be capitalized, but not 'money')